### ISSUE TITLE
Feature request: ADIN1110 support (IDFGH-12920)

### ISSUE THREAD

**Issue by stintel** [espressif/esp-eth-drivers#31]
Will you consider adding support for the ADIN1110 SPE chip?
https://www.analog.com/en/products/adin1110.html

IEEE 802.3cg SPE with PoDL and 1000m cable length sounds extremely interesting for IoT.

**Comment by kostaond** [comment]
@stintel, do you really need 1000m? Or are you interested in SPI interface in this particular case more? We currently support LAN867x which is 10BASE-T1S. And we currently planning to stick with it. The 10BASE-T1S is able to reach more than 100m in reality (standard's 25m is minimum maxima). This was discussed with Microchip representative and we also performed a test by ourselves in our office. The advantage of 10BASE-T1S is support of multidrop network topology which can be interesting in IoT since you don't need switches and so reduce the price and complexity of cabling.

What I'm considering though is adding support of 10BASE-T1S MAC-PHY device with SPI interface. Probably LAN8650 since I expect the PHY layer control will be exactly the same as LAN867x. It could be interesting for our customers because only ESP32 and ESP32P4 have RMII.

**Comment by stintel** [comment]
@kostaond I don't really need 1000m, but the [SparkFun MicroMod Single Pair Ethernet kit](https://www.sparkfun.com/products/24804) uses this chip. The kit makes it very easy to experiment with 802.3cg.

**Comment by kostaond** [comment]
@stintel, I see. However, Microchip also offers a module you can experiment with. Please see https://www.microchip.com/en-us/development-tool/EV06P90A We actually have used those for our experiments. 10BASE-T1L is point to point therefore it's quite straight forward. On the other hand, 10BASE-T1S offers multidrop so it could be more fun to experiment. However, it depends on your use case...  The only drawback is these modules are usually ridiculously expensive. 

To sum up,  we are not planning to add support of ADIN1110 in near future. I'll keep this issue open though and let people to vote to see if there is interest in supporting this chip. Thank you for understanding.
